core/templates/dev/head/filters.js

```diff
@@ -18,6 +18,8 @@
 
 oppia.constant('RULE_SUMMARY_WRAP_CHARACTER_COUNT', 30);
 
+oppia.constant('FEEDBACK_SUBJECT_MAX_CHAR_LIMIT', 50);
```

Comment:
Use assets/constants.js for constants that are shared between frontend and backend. The backend should read from this as well so that there's no skew.

Reply:
Done